paddle/fluid/operators/lookup_table_op.h

```diff
@@ -25,56 +28,88 @@ namespace operators {
 using Tensor = framework::Tensor;
 using LoDTensor = framework::LoDTensor;
 using SelectedRows = framework::SelectedRows;
+using DDim = framework::DDim;
+
+static constexpr int64_t kNoPadding = -1;
```

Comment:
`static` means that this variable is not exposed and is often used in .cc files. Why do we need static in this header file?